web/ui/module/lezer-promql/README.md

```diff
@@ -0,0 +1,43 @@
+# lezer-promql
+
+## Overview
+
+This is a PromQL grammar for the [lezer](https://lezer.codemirror.net/) parser system. It is inspired by the initial
+grammar coming from [Prometheus](https://github.com/prometheus/prometheus/blob/main/promql/parser/generated_parser.y)
+written in yacc.
+
+This library is stable but doesn't provide any guideline of how to use it as it has been integrated
+into [codemirror-promql](https://github.com/prometheus/prometheus/blob/main/web/ui/module/codemirror-promql). If you
+want to use this library, you perhaps want to actually use **@prometheus-io/codemirror-promql** instead.
+
+**Note**: This library is following the changes of the upstream grammar (i.e. coming from Prometheus) as closed as
```

Comment:
```suggestion
**Note**: This library is following the changes of the upstream grammar (i.e. coming from Prometheus) as closely as
```

Reply:
I'm wondering if this note make sense since lezer-promql is in prometheus and any changes of PromQL is changing as well this lib in the same PR.

Reply:
Yeah, the wording could be changed a bit, like:

"This library is a lezer-based implementation of the [authoritative, goyacc-based PromQL grammar](https://github.com/prometheus/prometheus/blob/main/promql/parser/generated_parser.y). Any changes to the authoritative grammar need to be reflected in this package as well."

Reply:
nice, I will change by that. Thanks !